certification/internal/shell/imagesource_registry_test.go

```diff
@@ -0,0 +1,30 @@
+package shell
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+)
+
+var _ = Describe("imageSourceRegistry", func() {
+	var (
+		imageSourceRegistryCheck ImageSourceRegistryCheck
+	)
+	Describe("Checking for valid image source registry ", func() {
+		Context("When the image source is in the approved registry", func() {
+			It("should pass Validate", func() {
+				ok, err := imageSourceRegistryCheck.Validate("registry.access.redhat.com/ubi8/ubi")
```

Comment:
This ends up being an integration test, not a unit test, if you are using actual images. Does this still pass if you pass it "dummy/image"?

Reply:
Brad, It's challenging for me to write test pertaining to the policies. Just can't wrap my head around it. Can you give me a quick lesson on it. It would improve my speed on designing these tests. Thank you. :-)